var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -69,32 +63,39 @@ class Hdf5(AutotoolsPackage):
     variant('java', default=False, description='Enable Java support')
     variant('threadsafe', default=False,
             description='Enable thread-safe capabilities')
-
+    variant('tools', default=False, description='Enable build tools')
```

Comment:
Isn't this the utilities(tools) like h5dump,h5diff, etc? Actually build here is a verb, so "Enable building the tools" would be better.

Reply:
I hope h5dump is enabled by default! 😰

Reply:
> It won’t be with ‘tools’, default=False.  Changing the default to True will build all the tools.  I don’t think we have any objection to them being built by default.  The build size will be larger, but I suppose anyone who cares can specify ‘~tools’.
It should definitely be on by default, for consistency with the autotools version of the package (and presumably with the precompiled binaries). Before merging I'll do a recursive diff of before-and-after with autotools and hdf5 versions...

Reply:
Also I noticed the tests for this package assume the tools are installed.